Sources/Templates/AutoRegisterable.swift

```diff
@@ -69,6 +69,32 @@ enum AutoRegisterable {
         lines.append("}")
         lines.append(.emptyLine)
 
+        let sortedSkeletons: [(Type, Type)] = types.classes
```

Comment:
Maybe `type.classes + type.structs` to make sure we're not constrained to just classes for UseCases.

Reply:
Good point, done! ✔️